From: Ben Collins <bcollins@ubuntu.com>
Date: Fri, 11 Nov 2005 20:01:26 +0000 (-0500)
Subject: [UBUNTU:scsi] Add basic hotplug support to fdomain
X-Git-Tag: v2.6.15-rc1
X-Git-Url: http://kernel.org/git/?p=linux/kernel/git/bcollins/ubuntu-2.6.git;a=commitdiff;h=e82633eb19c4b390846a6eecd72204b696448954

[UBUNTU:scsi] Add basic hotplug support to some drivers

UpstreamStatus: Unsubmitted

Signed-off-by: Ben Collins <bcollins@ubuntu.com>
---

--- a/drivers/scsi/fdomain.c
+++ b/drivers/scsi/fdomain.c
@@ -1720,6 +1720,15 @@ static int fdomain_16x0_release(struct S
 	return 0;
 }
 
+#ifndef PCMCIA
+static struct pci_device_id fdomain_pci_tbl[] __devinitdata = {
+	{ PCI_VENDOR_ID_FD, PCI_DEVICE_ID_FD_36C70,
+	  PCI_ANY_ID, PCI_ANY_ID, 0, 0, 0UL },
+	{ }
+};
+MODULE_DEVICE_TABLE(pci, fdomain_pci_tbl);
+#endif
+
 struct scsi_host_template fdomain_driver_template = {
 	.module			= THIS_MODULE,
 	.name			= "fdomain",
